src/devices/bus/coco/coco_t4426.cpp

```diff
@@ -182,7 +185,9 @@ ROM_START( coco_t4426 )
 
 	// 8 banked ROM:s
 	ROM_LOAD("tercoED4426-0-8549-5.3.bin",  0x0000, 0x2000, CRC(45665428) SHA1(ff49a79275772c4c4ab1ae29db662c9b10a744a7))
-	ROM_LOAD("tercoED4426-1-8549-5.3.bin",  0x2000, 0x2000, CRC(44baba33) SHA1(01cee1b208c158e598e7ecd2189b5e0ffa7f3ab9))
+	//ROM_LOAD("tercoED4426-1-8549-5.3.bin",  0x2000, 0x2000, BAD_DUMP CRC(44baba33) SHA1(01cee1b208c158e598e7ecd2189b5e0ffa7f3ab9))
```

Comment:
Why keep the known bad dump here?  We have source control for finding changes.  Is the bad dump useful for anything?

Reply:
I was thinking keeping it as a reference as the ROM hit distributions already but I will remove the reference and rely on source control for that.